### ISSUE TITLE
split of Brain Tumor dataset

### ISSUE THREAD

**Issue by Zolento** [282857341/nnFormer#75]
UNETR的原文是这样描述的：“For the brain tumor segmentation task, the entire training set of 484 multi-modal multi-site MRI data (FLAIR, T1w, T1gd, T2w) with ground truth labels of gliomas segmentation necrotic/active tumor and oedema is utilized for model training.”
而nnformer对于这个的描述是:"For the split of data, we follow the instruction of UNETR, where ratios of training/validation/test sets are 80%, 15% and 5%"
在你们的源码里，以及提供的dataset.json文件里，提供的split方法和后者一致。而我并没有在UNETR中找到他们关于划分的描述或者文件(可能我遗漏了)，而且UNETR所报告的表现更像是在整个训练集上训练得到的结果，因此我想知道这种划分方式是从哪来的。

**Comment by 282857341** [comment]
''For brain and spleen segmentation tasks in MSD dataset, we split the data into training, validation and test with a ratio of 80:15:5.''
在4.3. Implementation Details 的最后有这么一句话

**Comment by Zolento** [comment]
刚刚看到了，感谢回答。
